### DISCUSSION TITLE
Monitor BenQ GW2750H

### DISCUSSION THREAD

**Discussion by peminator** [MonitorControl/MonitorControl#1273] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Brightness controls works, volume controls appear only to work (meaning i can see the volume conrol and change its value), but the real volume on monitor is not changed. Tested on Windows machine sunin gsame hdmi cable, volme changing works)

### Expected behavior

installed freshly downloaded version, changed volume, did not change the volume

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.6.1 Monterey 
- Mac model: Macbook (12", Early 2016)
- MonitorControl version: 4.1.0
- Monitor(s): BenQ GW2750H
- Monitor Cable(s)/Connection(s): MacBook usbc -> dock with pd,1xUsb,hdmi port -> monitor
- Apple Silicon/M1 (yes or no): no
```

**Comment by bestlufeng** [comment]
I have the same phenomenon in my Redmi 4K  monitor,  it work with DDC mode

- macOS version: 12.6.1 Monterey 
- Mac model: macbook m1 pro 14
- MonitorControl version: 4.1.0
- Monitor(s): Redmi 4K
- Monitor Cable(s)/Connection(s):  typc by display mode
- Apple Silicon/M1 (yes or no): yes

**Reply by Louis296** [reply]
i have the same phenomenon too, also use the Redmi 4K monitor.

**Comment by waydabber** [comment]
@peminator - are you sure DDC brightness control fully works, not only the software dimming part of the scale (by default 0-50%)? if so, then DDC volume should work as well. Other possibility is that the display does not support DDC volume control (many displays fail with this). Windows does not use DDC to control volume but simply alters the digital audio level via its software mixer (macOS does not do that).

**Reply by peminator** [reply]
hi, now on v 4.1.0, macbook air 2018, Osx Ventura, brightness really seems to work in range 0-50%, putting higher brightness does not seem to do anything. Volume slider can be moved, but does not change anything. Anything else i can do?
